.github/workflows/ompi_mpi4py.yaml

```diff
@@ -32,6 +32,7 @@ jobs:
               --disable-man-pages
               --disable-mpi-fortran
               --disable-oshmem
+              --enable-debug
```

Comment:
Can you add `--enable-mem-debug` as well?

Reply:
done